<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materielm</class>
 <widget class="QDialog" name="materielm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>etat</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton #pushButton{
	
	background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton #pushButton:hover{
	
	background-color: rgb(98, 185, 203);
}
QPushButton #pushButtonp:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(98, 185, 203);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(27, 75, 101);
</string>
    </property>
    <property name="text">
     <string>Formulaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>dispo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomm">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dispom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouterm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prixm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 75, 101);</string>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_suppm">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_supp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qrcode">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>50</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="horizontalFrame">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>911</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Liste des materiels</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pdfm">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	border-image: url(:/pdf.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>80</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tab_mat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rechercherm">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;
border-radius:5px;
color: rgb(27, 75, 101);
</string>
    </property>
    <property name="placeholderText">
     <string>rechercher</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="codeqr">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>code </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_trimat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_trimat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>idmat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nommat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prixmat</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="desc">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>desc</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="asc">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>ASC</string>
    </property>
   </widget>
  </widget>
  <zorder>label_7</zorder>
  <zorder>label_10</zorder>
  <zorder>widget</zorder>
  <zorder>label_8</zorder>
  <zorder>pb_suppm</zorder>
  <zorder>comboBox_supp</zorder>
  <zorder>label_11</zorder>
  <zorder>label_qrcode</zorder>
  <zorder>horizontalFrame</zorder>
  <zorder>widget_2</zorder>
  <zorder>codeqr</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
